src/translations/pages/contact.ts

```diff
@@ -1550,9 +1565,54 @@ export const Contact = {
         },
       ] as RefundReason[],
       errorMessages: {
-        empty: _('Område mangler', 'Area is missing', 'Område mangler'),
+        empty: _(
+          'Grunn for refusjon mangler',
+          'Reason for refund is missing',
+          'Grunn for refusjon mangler',
+        ),
       },
     },
+
+    ticketType: {
+      tickets: [
+        {
+          id: 'singleTicket',
+          name: _('Enkeltbillett', 'Single ticket', 'Enkeltbillett'),
+        },
+        {
+          id: 'periodTicket',
+          name: _(
+            'Periodebillet (sonebasert)',
+            'Period ticket (zone-based)',
+            'Periodebillet (sonebasert)',
+          ),
+        },
+        {
+          id: 'framYoung',
+          name: _('FRAM Ung', 'FRAM Young', 'FRAM Ung'),
```

Comment:
At the moment, all of the FRAM tickets have the same name in any language in the app at least, should that be the case here as well?

Reply:
I´ll update the naming👍